WordPress/wordpress-playground
# Website Query API: Add `import-site` and `import-content` options

## What is this PR doing?

Adds query parameter `import-site` and `import-content` to the Playground website's Query API.

- `import-site` - Imports site files and database from a **zip file** specified by URL.
- `import-content` - Imports site content from a **WXR or WXZ file** specified by URL.

## What problem is it solving?

Resolves #608.

- #608

## Testing Instructions

- Start local server with `npm run dev`
- Visit the following URL and confirm **site files and database** have been imported:

  http://localhost:5400/website-server/?php=8.0&wp=6.4&storage=none&import-site=https://raw.githubusercontent.com/dmsnell/playground-experiments/trunk/just-a-little-change/playground.zip

- Visit the following URL and confirm **site content** has been imported:

  http://localhost:5400/website-server/?php=8.0&wp=6.4&storage=none&networking=yes&import-content=https://raw.githubusercontent.com/WordPress/theme-test-data/master/themeunittestdata.wordpress.xml
